### DISCUSSION TITLE
Sagetv client on a Ubuntu VM on Unraid

### DISCUSSION THREAD

**Discussion by gdippel** [OpenSageTV/Community#107] [Linux]
I've installed the client on my Ubuntu 22.04 VM but while I can get video, I have no sound.  My encoders are OTA via HDhomerun and other channels via Channels DVR.  My server is on a docker on unRaid.  The mini client works fine on my Amazon firestick.  I've tried various sound configurations for Mplayer but no luck.  Has anyone gotten the client to work on a Ubuntu VM?

**Comment by wayner9** [comment]
I haven't tried installing the client in a VM, but does sound work with other applications in the VM?  In other words, are you sure it is a SageTV issue and not a general hardware config issue in the VM?

I have never used the client in Linux as I have lots of HD300 boxes that are sufficient, and I am increasingly using stuff like Plex or Jellyfin to playback media.

**Reply by gdippel** [reply]
Sound works in the VM, I suspect it's sage's implementation of Mplayer that causes the problem.  I'll continue to use Sagetv on my Firestick and HD 300.
